packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -105,4 +109,48 @@ export const Examples = () => {
   );
 };
 
+export const Basic: ComponentStory<typeof PanelChrome> = (args: PanelChromeProps) => {
+  const theme = useTheme2();
+
+  const contentStyle: CSSProperties = {
+    background: theme.colors.background.secondary,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  };
```

Comment:
I think we should still put this in a different function just so we don't have two different styles that are meant to do the exact same, something like:
```
const getContentStyle = (theme: GrafanaTheme2) => {
  return {
    background: theme.colors.background.secondary,
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  }
}
```

And then call `const contentStyle: CSSProperties = getContentStyle(theme);` from both places this is used